tests/ttnn/unit_tests/gtests/tensor/test_create_tensor.cpp

```diff
@@ -13,7 +13,7 @@
 
 #include "ttnn_test_fixtures.hpp"
 
-void run_create_tensor_test(tt::tt_metal::Device* device, ttnn::Shape input_shape) {
+void run_create_tensor_test(tt::tt_metal::Device* device, ttnn::SimpleShape input_shape) {
```

Comment:
nit (in tests): const ttnn::SimpleShape&

Reply:
Brian reported that const& there ends up with a segfault when accessing shape. I don't want to explore this now. But will revisit as we will add lots of tests here